apstra/api_utils.go

```diff
@@ -0,0 +1,23 @@
+package apstra
+
+import (
+	"encoding/json"
+	"fmt"
+	"reflect"
+)
+
+func jsonEqual(m1, m2 json.RawMessage) (bool, error) {
+	var map1 interface{}
+	var map2 interface{}
+
+	var err error
+	err = json.Unmarshal(m1, &map1)
+	if err != nil {
+		return false, fmt.Errorf("error unmarshalling string 1 : %s", err.Error())
```

Comment:
If we're going to return an error here we should use the error wrap pattern `%w` instead of `%s`.

But I'd still prefer to see this function move into `test_helpers.go` and follow the `compareSlices()` pattern.